### DISCUSSION TITLE
Adding support for llm-ls

### DISCUSSION THREAD

**Discussion by McPatate** [zed-industries/zed#6790] [Feature Requests]
Hi Zed team, super cool to hear you open sourced Zed!

I work on https://github.com/huggingface/llm-ls & our extensions at Hugging Face and was wondering what your vision for embedding AI into Zed was.

Are you going towards the plugin/extension route where people contribute their own or do you have an opinionated list of pre-installed extensions?

Let me know if you'd be interested in collaborating on this topic, I definitely am! 🦀

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
